pipeline/config.go

```diff
@@ -38,6 +38,12 @@ func (c Config) BuildOperators(bc operator.BuildContext) ([]operator.Operator, e
 		}
 		operators = append(operators, op...)
 	}
+
+	if defaultOperator != nil {
+		bc.DefaultOutputIDs = []string{defaultOperator.ID()}
```

Comment:
BuildContext is not used after this point, so this line can be removed.

Reply:
fixed